tests/unit/jobs/test_views.py

```diff
@@ -841,3 +862,47 @@ def test_get_submission_by_pk_when_user_is_neither_challenge_host_nor_submission
         response = self.client.get(self.url)
         self.assertEqual(response.data, expected)
         self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
+
+
+class ChallengeLeaderboardTest(BaseAPITestClass):
+
+    def setUp(self):
+        super(ChallengeLeaderboardTest, self).setUp()
+
+        self.submission = Submission.objects.create(
+            participant_team=self.participant_team,
+            challenge_phase=self.challenge_phase,
+            created_by=self.user1,
+            status="finished",
+            input_file=self.challenge_phase.test_annotation,
+            method_name="Test Method",
+            method_description="Test Description",
+            project_url="http://testserver/",
+            publication_url="http://testserver/",
+        )
+
+        self.submission.is_public = True
+        self.submission.status = Submission.FINISHED
+        self.submission.save()
+
+        # NOTE: result schema has one key missing for test_leaderboard_schema_key_missing
+        self.result_json = {
+            'score': 0
+        }
+
+        self.leaderboard_data = LeaderboardData.objects.create(
+            challenge_phase_split=self.challenge_phase_split,
+            submission=self.submission,
+            leaderboard=self.leaderboard,
+            result=self.result_json
+        )
+
+    def test_leaderboard_schema_key_missing(self):
```

Comment:
Please change `test_leaderboard_schema_key_missing` to `test_leaderboard_schema_with_missing_key`